### ISSUE TITLE
Paper replication

### ISSUE THREAD

**Issue by shakibyzn** [MLSLT/MLSLT#1]
Hi,

I'm trying to replicate the same results using the proposed model and the SP-10 dataset. For the video encoding part, I'm using the `timm/tf_efficientnet_b7.ra_in1k` model with the below stat almost like paper: 
train: 8297 dev:1420 test: 2021 

With the same config.yml in the repo, I only reached BLEU-4 of 0.24 on the validation set!

Could you help me with this? or provide me with the train/dev/test dataset objects?

**Comment by hahaha0528** [comment]
Hello,
I noticed you’ve faced the same issue—may I ask if you’ve already solved it（BLEU-4：0.94，ROUGE：5.61）?
If so, could you kindly share the steps? I’d really appreciate your help.



> Hi,
> 
> I'm trying to replicate the same results using the proposed model and the SP-10 dataset. For the video encoding part, I'm using the `timm/tf_efficientnet_b7.ra_in1k` model with the below stat almost like paper: train: 8297 dev:1420 test: 2021
> 
> With the same config.yml in the repo, I only reached BLEU-4 of 0.24 on the validation set!
> 
> Could you help me with this? or provide me with the train/dev/test dataset objects?

**Comment by shakibyzn** [comment]
Hi,

Unfortunately not. However, I pre-trained their model on Ph14T, How2Sign, and CSL-Daily and fine-tuned it again on three subsets of SP-10 (DGS, CSL, BSL). The results were close to what they reported in the paper. 

However, it took some time. I suggest you pick another multilingual dataset like youtube-sl-25.

**Comment by hahaha0528** [comment]
This dataset seems really large. Okay,I'll try out the suggestions you provided. Thank you !






💤
***@***.***

**Comment by shakibyzn** [comment]
Yes. You can also choose to download those smaller subset languages. I think half of the dataset is ASL. You can use [yt-dlp](https://github.com/yt-dlp/yt-dlp) for it.
